static/bidder-info/melozen.yaml

```diff
@@ -0,0 +1,17 @@
+# We have the following regional endpoint domains: us-east and us-west
+# Please deploy this config in each of your datacenters with the appropriate regional subdomain
+endpoint: "https://prebid.melozen.com/rtb/v2/bid?publisher_id={{.PublisherID}}"
+endpointCompression: gzip
+geoscope:
+  - global
+maintainer:
+  email: DSP@meldong.com
```

Comment:
Prebid team has sent email to verify above maintainers email. Requesting to responding back on email thread

Reply:
@benben2001, email bounced back. Requesting to confirm maintainer email

Reply:
Thank you for pointing out the error with the email address. Indeed, there was a typo in the email provided. The correct email should be DSP@melodong.com. I have updated the static/bidder-info/melozen.yaml file with the correct information and pushed the changes to the PR. Please review the updated changes at your convenience. Thank you for your patience and assistance.

Reply:
Sent email again to the correct address.

Reply:
Hi @gargcreation1992 ,

I have sent the "received" confirmation to the specified email address as requested. Please let me know if there's anything else needed from my side.

Thank you for your assistance.

Best,
MeloZen

Reply:
received response from [DSP@melodong.com](mailto:DSP@melodong.com)